lib/node_modules/@stdlib/math/base/special/sincospi/docs/repl.txt

```diff
@@ -0,0 +1,37 @@
+
+{{alias}}( [out,] x )
+    Simultaneously computes the sine and cosine of a number times π.
+
+    Parameters
+    ----------
+    out: Array|TypedArray|Object (optional)
+        Destination array.
+
+    x: number
+        Input value.
+
+    Returns
+    -------
+    y: Array|TypedArray|Object
+        Sine and cosine (in radians).
```

Comment:
Should this be changed?

Reply:
Ohh, the `in radians` part. I sorta take issue with that as confusing in the first place since it's the *parameter* that's in radians, not the return value.

Reply:
In this case though, it seems like that should just be removed. It's the input that's in multiples of pi, which might a better way to communicate that.

Reply:
@athan, can you weigh in, especially also about the radians part, which affects many trig packages?

Reply:
How about `a two element array containing sin(πx) and cos(πx)`

Reply:
Yes, this is incorrect (along with other trig packages). The `(in radians)` qualifier should be moved to the description of `x`.

Reply:
Think this would be an improvement, Ricky, but would remove the article.

Reply:
I don't mind updating the function signature descriptions in the REPL docs and move the `(in radians)` part to the description of the input value. Unless someone else wishes to.

Reply:
Sounds good to me. Thanks, @Planeshifter!